controller/invitation.go

```diff
@@ -63,7 +63,18 @@ func (c *InvitationController) CreateInvite(ctx *app.CreateInviteInvitationConte
 		})
 	}
 
-	err = c.app.InvitationService().Issue(ctx, currentIdentity.ID, ctx.InviteTo, invitations)
+	var redirectOnSuccess, redirectOnFailure string
+	links := ctx.Payload.Links
+	if links != nil {
+		if links.OnSuccess != nil {
```

Comment:
@dipak-pawar  It is therefore possible for the redirect URLs to not have been specified. How are we handling this in `Accept` Invitation in that case?

Reply:
@sbose78 if a user is not specifying redirect URLs, then we'll be redirecting them to default URLs configured in auth using config map for key `invitation.accepted.url`

Reply:
@sbose78 Added test for it in 0e23dad0bf7204cbaaecdd4f862fd4a7462b4d0b